<launch>
    <node pkg="track_demo" type="tracker.py" name="tracker" output="screen">
        <param name="camera_topic" value="/prophesee/event_frame" />
        <param name="camera_info" value="" />
        <!-- $(find track_demo)/config/camera_config/camera_param_gazebo_monocular.yaml -->
        <param name="tracker_conf" value="" />
    </node>
</launch>